test/torchaudio_unittest/common_utils/ctc_decoder_utils.py

```diff
@@ -0,0 +1,32 @@
+from torchaudio.models.decoder import CTCDecoderLM, CTCDecoderLMState
+from torchaudio.models.decoder._ctc_decoder import _create_word_dict, _KenLM, _load_words
+
+
+class CustomZeroLM(CTCDecoderLM):
+    def __init__(self):
+        CTCDecoderLM.__init__(self)
+
+    def start(self, start_with_nothing: bool):
+        return CTCDecoderLMState()
+
+    def score(self, state: CTCDecoderLMState, token_index: int):
+        return (state.child(token_index), 0.0)
+
+    def finish(self, state: CTCDecoderLMState):
+        return (state, 0.0)
+
+
```

Comment:
As a baseline, can we add some toy custom LM that's neither ZeroLM nor KenLM, that
- returns hardcoded scores for prefixed input pattern ?
- runs `torch.nn.Module` inside. (to be sure that such integration works without a software issue)